src/app/containers/CompanyForm.js

```diff
@@ -62,6 +76,23 @@ export class CompanyForm extends Component {
 						/>
 					</MuiPickersUtilsProvider>
 				</FormGroup>
+				<FormControl margin="normal" style={{display: "block", textAlign: "left"}}>
```

Comment:
i think after #11 is merged, you can remove this inline style :)

Reply:
I think so too!